webpack/ForemanTasks/Components/TaskDetails/TaskDetailsSelectors.js

```diff
@@ -30,7 +30,7 @@ export const selectErrors = state => {
 
 export const selectProgress = state =>
   selectTaskDetails(state).progress
-    ? parseFloat((selectTaskDetails(state).progress * 100).toFixed(2))
+    ? parseInt(selectTaskDetails(state).progress * 100, 10)
```

Comment:
The `parseInt` doesn't feel right (shouldn't it do `String -> Integer`?).

The number here should be 0 <= number <= 1 with precision to two decimals so multiplying it by 100 should yield an integer already. If we want to be sure we really get an integer percentage, wouldn't something like `Math.trunc` be better?

@laviro opinions on this?

Reply:
Yeah I agree, `Math.trunc` is exactly for that, to cut off the dot and the digits to the right of the number.
although multiplying by 100 should do the work in this case, it will "protect" us in cases where there's a decimal.